landoui/treestatus.py

```diff
@@ -211,15 +211,25 @@ def treestatus_tree(tree: str):
     """Display the log of statuses for an individual tree."""
     api = TreestatusAPI.from_environment()
 
-    logs_response = api.request("GET", f"trees/{tree}/logs")
+    try:
+        logs_response = api.request("GET", f"trees/{tree}/logs")
+    except LandoAPIError as exc:
+        if not exc.detail:
+            raise
+
+        error = f"Error received from LandoAPI: {exc.detail}"
+        logger.error(error)
+        flash(error, error)
+        return redirect(request.referrer, code=exc.status)
```

Comment:
This should be using `e.status_code`. `e.status` refers to something else. This will likely raise a different exception in the redirect.